pattern-categorization/innersource-program-mind-map.md

```diff
@@ -2,90 +2,106 @@
 
 ## Begin
 
-### [Start as an Experiment](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/patterns/2-structured/start-as-experiment.md)
+### Program Setup
 
-### [Dedicated Community Manager](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/patterns/2-structured/dedicated-community-leader.md)
+#### Management hesitates to invest in InnerSource
+
+##### [Start as an Experiment](https://patterns.innersourcecommons.org/p/start-as-experiment)
+
+#### Lack of credibility of the InnerSource program
```

Comment:
The dedicated community leader pattern seems to stress the importance of community building around a project. What do you think of something like `Poor InnerSource outcome when there's no community`?

Reply:
The pattern mentions
> If a community can not grow fast enough and pick up enough speed, chances are they won't be able to convincingly demonstrate the potential of InnerSource.

Seems like the slow pace is important here. So maybe:
`Slowly growing community hinders InnerSource`

Reply:
I like this.

Reply:
Done!